Packs/EmailCommunication/.secrets-ignore

```diff
@@ -1,5 +1,18 @@
 test1@gmail.com
 test3@gmail.com
 test2@gmail.com
+test4@gmail.com
 test123@gmail.com
-id:ii_kgjzy6yh0
\ No newline at end of file
+id:ii_kgjzy6yh0
+soc_sender@company.com
+end_user@company.com
+cc_user@company.com
+cc_user_1@company.com
+cc_user_2@company.com
+bcc_user@company.com
+BY5PR09ME5460A9F1D8E34A12904AE86EB6199@BY5VR02MB5660.namprd09.prod.outlook.com
+BY5PR03ME5460A9F1D8E34A12904AE86EB6191@BY5VR12MB5660.namprd09.prod.outlook.com
+listName=\"XSOAR
+AAMkAGRcOGZlZTEzLTkyZGDtNGJkNy1iOWMxLYM0NTAwODZhZjlxNABGAAAAAAAP2ksrJ8icRL4Zhadm7iVXBwAkkBJXBb0
+sRJWC0zdXEMqsAAAAAAEMAAAkkBJFBb0fRJWC0zdXEMqsABApcWVYAAA='
+dm7iVXBwAkkBJXBb0sRJWC0zdXEMqsAAAAAAEMAAAkkBJFBb0fRJWC0zdXEMqsABApcWVYAAA=
```

Comment:
Those are some email message IDs that are included in test data for unit test scripts.  They're not real, but are made to look like actual message IDs so that the test script validates the function works with real-looking data.